openid_connect.js

```diff
@@ -131,21 +131,21 @@ function validateIdTokenClaims(r, claims) {
     const missingClaims = requiredClaims.filter((claim) => !claims[claim]);
 
     if (missingClaims.length > 0) {
-        r.error(`OIDC ID Token validation error: missing claim(s) ${missingClaims.join(' ')}`);
+        r.error(`OIDC ID Token validation error for ` + r.headersIn['Host'] + r.uri + `: missing claim(s) ${missingClaims.join(' ')}`);
         return false;
     }
 
     // Check 'iat' validity
     const iat = Math.floor(Number(claims.iat));
     if (String(iat) !== claims.iat || iat < 1) {
-        r.error("OIDC ID Token validation error: iat claim is not a valid number");
+        r.error(`OIDC ID Token validation error for ` + r.headersIn['Host'] + r.uri + `: iat claim is not a valid number`);
```

Comment:
When I talk about style, this spot is a good example. Here you switched to template literals, but in #L168 you are using string concatenation under the same circumstances. Was this done for a reason I didn’t notice?